packages/babel-plugin-transform-es2015-classes/src/loose.js

```diff
@@ -6,6 +6,25 @@ export default class LooseClassTransformer extends VanillaTransformer {
   constructor() {
     super(...arguments);
     this.isLoose = true;
+    this.aliasInserted = false;
+    this.methodAlias = this.path.scope.generateUidIdentifier("proto");
```

Comment:
Nit: this uses a global name. We can instead test if `methodAlias` is set yet, and create it when needed.